### PR TITLE
chore: Switch from edx-sphinx-theme to sphinx-book-theme

### PR DESCRIPTION
**Pull request by xitij2000**
The edx-sphinx theme is being deprecated, and replaced with sphinx-book-theme. This removes references to the deprecated theme and replaces them with the new standard theme for the platform.

**Testing instructions:** 
- Run `make docs` and see that the docs are generated properly and use the sphinx-book-theme

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @xitij2000!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by mphilbrick211** [comment]
@xitij2000 - looks like there's some failing tests with this one.

**Comment by mphilbrick211** [comment]
Hi @xitij2000 - is this all set for review?

**Comment by Cup0fCoffee** [comment]
:+1: Assuming the conflicts are going to be resolved.

- [x] I tested this: ran `make docs`
- [x] I read through the code

**Comment by mphilbrick211** [comment]
@xitij2000 - would you mind fixing the branch conflicts?

@kashifch - this is ready for your review/merge. Thanks!

File: docs/conf.py [hunk 1]
```diff
@@ -96,15 +95,45 @@
...
+                src="https://i.creativecommons.org/l/by-sa/4.0/80x15.png"/>
+        </a>
+        <br>
+        These works by
+            <a
+                xmlns:cc="https://creativecommons.org/ns#"
+                href="https://openedx.org"
+                property="cc:attributionName"
+                rel="cc:attributionURL"
+            >The Center for Reimagining Learning</a>
```
Comment (feanil) [inline docs/conf.py]:
```suggestion
            >The Axim Collaborative</a>
```

**Review by feanil — APPROVED** [review]

**Comment by mphilbrick211** [comment]
Friendly ping on this, @kashifch! :)

**Review by feanil — APPROVED** [review]

**Comment by feanil** [comment]
@xitij2000 it looks like this needs a rebase before it can be merged.  Can you do that?

**Comment by openedx-webhooks** [comment]
@xitij2000 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/course-discovery&entry.867055334=https://github.com/openedx/course-discovery/pull/3853&entry.1484655318=https://github.com/xitij2000&entry.752974735=2023-03-24+07:30&entry.1917517419=2023-04-21+17:26&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**